### ISSUE TITLE
Remove harmony project file from the library

### ISSUE THREAD

**Issue by RavanJAltaie** [openzim/zim-requests#762]
Please remove this file from the library
https://download.kiwix.org/zim/zimit/harmony-project.org_en_all_2023-09.zim

**Comment by Jaifroid** [comment]
@RavanJAltaie What's the reason for wanting it removed?

**Comment by RavanJAltaie** [comment]
@Jaifroid the resulted file had some problems, I've disabled the recipe and the issue is being troubleshooting here
https://github.com/openzim/zimit/issues/258

**Comment by Jaifroid** [comment]
@RavanJAltaie Yes, but as I wrote in that issue, the file is not actually broken, and works fine (as fine as any other Zimit file, which can be fragile) in Kiwix JS and Kiwix PWA. The issue is more on the side of Kiwix Serve, I believe. I've tested this file extensively, and it isn't broken. In fact, I would say that it's valuable to keep a file (or move it to dev) if it's a file that shows up issues in one implementation and not in others, as it provides a good test.

**Comment by RavanJAltaie** [comment]
@Jaifroid I thought the same as you in the beginning, that why I moved the file to the library as I though the content is there. But then we've decided to remove it and post the perfect version once ready
@Popolechien shall we keep it?

**Comment by Jaifroid** [comment]
Well, maybe this little demo will convince you both that this ZIM is good! 😉

https://github.com/openzim/zim-requests/assets/4304337/469da10d-90d3-4d8b-b6dd-4a1805ee7a33

As I say, the issue is with the app(s) doing the playback, not with the ZIM itself. Moreover, I've been showcasing this ZIM in imagery for the new Zimit support of the PWA, so it would be a shame to remove it. Keeping this ZIM doesn't stop you working on an even more perfect version. 🍀

**Comment by Jaifroid** [comment]
So, might we close this issue as not planned? As I said, keeping it doesn't stop you working on further versions.

**Comment by RavanJAltaie** [comment]
@Jaifroid agreed!

**Comment by Popolechien** [comment]
Sorry for being late to the party. I agree it would be a shame to get rid of it if it works, but I was under the impression that kiwix-serve would be our standard for demoing content. I'm kind or wary of distributing a file that would neither work on server/raspberry nor on desktop. What about android /iOS?

@kelson42 Thoughts?

**Comment by Popolechien** [comment]
Ok, no luck on Android either - I'd remove it from the main library onto dev.

**Comment by Jaifroid** [comment]
Um, as I keep saying, it's working fine on Kiwix Serve! See https://library.kiwix.org/viewer#harmony-project.org_en_all_2023-09/A/www.harmony-project.org/music-library . The only issue is you need to adjust your browser window size to (roughly) the size of the initial scrape to see **some** of the images. That's not a problem with the scraper, nor really a problem with the ZIM.

**Comment by Popolechien** [comment]
> The only issue is you need to adjust your browser window size to (roughly) the size of the initial scrape to see some of the images

There is no word to describe how poor and unenforceable UX that is :-)

**Comment by Jaifroid** [comment]
LOL, but if what you're actually interested in is the sheet music contained in the ZIM (the main point of the ZIM), then that's just cosmetic. The sheet music is all there, and downloadable from the ZIM. I think that this is as good as it gets with the current Zimit technology. Zimit doesn't have a way to run at multiple different browser window sizes, so it can't scrape different image sets that users' browsers might request, only what was available to it at run time.

Bottom line, IMO, is that the ZIM is a valuable resource and perfectly useable, issues with it are cosmetic. By all means try to improve that, but don't nuke this one in the meantime. 😊

**Comment by Jaifroid** [comment]
(I think it's time to found a Swiss Society for the Prevention of Cruelty to ZIM Archives 🤣)

**Comment by Jaifroid** [comment]
OK, but I really doubt you can improve the UX issue with current Zimit software, because there is no option to change resolution during scrape, which is what would be needed. Hopefully I'm wrong.

**Comment by benoit74** [comment]
Done
